packages/core/src/components/axes/two-dimensional-axes.ts

```diff
@@ -71,13 +70,18 @@ export class TwoDimensionalAxes extends Component {
 			// To be able to tell the final width & height of the axis when initiaing the transition
 			// The invisible axis is updated instantly and without a transition
 			const invisibleAxisRef = child.getInvisibleAxisRef();
-			const { width, height } = DOMUtils.getSVGElementSize(invisibleAxisRef, { useBBox: true });
+			const {
+				width,
+				height
+			} = DOMUtils.getSVGElementSize(invisibleAxisRef, { useBBox: true });
```

Comment:
```suggestion
			const { width, height } = DOMUtils.getSVGElementSize(invisibleAxisRef, { useBBox: true });
```
would rather this on one line